src/main/java/me/crazybanana/events/commands/event.java

```diff
@@ -45,11 +45,19 @@ else if(args.length == 3 && args[0].equalsIgnoreCase("setup") && args[1].equalsI
         }
 
         // **Host Command**
```

Comment:
Create new function for event kick, revive and end

Reply:
Reminder: And leave